hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsReadFooterMetrics.java

```diff
@@ -47,6 +52,20 @@
 public class ITestAbfsReadFooterMetrics extends AbstractAbfsScaleTest {
 
   public ITestAbfsReadFooterMetrics() throws Exception {
+    checkPrerequisites();
+  }
+
+  private void checkPrerequisites(){
+    checkIfConfigIsSet(FS_AZURE_METRIC_ACCOUNT_NAME);
```

Comment:
We have a method in base class already defined for this purpose. We can use that itself `assumeValidTestConfigPresent`

Reply:
Taken

Reply:
nit: Here also we could have used the `assumeValidTestConfigPresent` method